Set up basic CSS classes and structured Bio-section

camelPhonso opened the pull request:
relates #2 I've altered some of the original naming conventions so it's worth you looking over all the classes, particularly with the css editing of each section you'll see I changed it to be a bit more inline with BEM. The bio section currently has random online pictures, we can change those as well of course. I also have .bordered on some elements right now as a guideline as I work on things but will remove that later on

camelPhonso committed:
Set up basic CSS classes and structured Bio-section

relates #2 I've altered some of the original naming conventions so it's worth you looking over all the classes, particularly with the css editing of each section you'll see I changed it to be a bit more inline with BEM. The bio section currently has random online pictures, we can change those as well of course. I also have .bordered on some elements right now as a guideline as I work on things but will remove that later on

zakkariyaa reviewed (changes requested):
Hey dude. couple things:

- can you define CSS classes for all layouts (flex, row, column, left, right, center, stack-xl, stack-l.., wrap, nowrap, gap-xl, gap-l..)

- box-sizing:border-box; is repeated twice

camelPhonso committed:
requested changes

Relates #2 Deleted duplicate box-sizing and added classes for .flex, .grid, .justify-start, .justify-end and .stack. I've not added any others because I think this page won't require too much css beyond what we have, but we can tweak further as we go.
